### ISSUE TITLE
ServerCompleteAuthorizeResponse - shouldn't auto send response

### ISSUE THREAD

**Issue by Eyewash01** [thephpleague/omnipay-sagepay#37]
It would be better if you could grab the output generated from the sendResponse method, as opposed to it automatically echo-ing it out and exiting.

For example, a framework may have various tasks it needs to complete after each request, and shouldn't just be "exit"-ed.

I would suggest a new method called generateResponse - which which does the same but returns the output instead of sending it.

**Comment by AndrewCarterUK** [comment]
I had the same problem and I've added an extra parameter to the methods. The pull request probably will not get accepted but you're welcome to use the branch I modified.

I'll leave repository up after the pull request is accepted/closed BUT... if I were you I would fork my patch of the repository yourself so that you have complete control.

In your composer.json:

```json
{
    "require": {
        "omnipay/sagepay": "dev-AndrewCarterUK-patch-1"
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/AndrewCarterUK/omnipay-sagepay"
        }
    ]
}
```

**Comment by judgej** [comment]
Fixed in PR #69 - the `acceptNotification` handler will output the response, which can be caught if necessary using `ob_start`. By default it will exit when outputting the final result, but there is an option to disable that, so you can continue processing if you want.
